plugins/mcreator-localization/lang/texts.properties

```diff
@@ -882,6 +884,7 @@ elementgui.common.event_right_clicked_air=When right clicked in air (player loc.
 elementgui.common.event_right_clicked_block=When right clicked on block (hand loc.))

```

Comment:
Please remove one ")" at the end.

Reply:
Am I the only one who don't see the `)` in-app if I don't put two `))` at the end of this line?

Reply:
Mac sometimes does not render the last characters of text on Mac for some unknown reason. 

There is definitely no specification for Java properties files requesting escaping ) as )).

Reply:
Remove other cases of this too please.

Reply:
Do not unresolve this. Most users see it like this:

![image](https://user-images.githubusercontent.com/16374228/95770967-86cd8b80-0cba-11eb-8127-bdf1ca319760.png)